httpx/models.py

```diff
@@ -514,12 +514,12 @@ def encode_data(
         self, data: dict = None, files: RequestFiles = None, json: typing.Any = None
     ) -> typing.Tuple[bytes, str]:
         if json is not None:
-            content = jsonlib.dumps(json).encode("utf-8")
+            content = jsonlib.dumps(json).encode()
```

Comment:
Let's revert the all changes related to utf-8. I think what we were doing before is correct.

Reply:
sure I reverted it, is the reason "explicit is better than implicit"?

Reply:
Calling `.encode()` without a parameter isn't the same as with utf-8. We definitely want to be able to encode international characters to utf-8 in these scenarios.

Reply:
Ah sorry, I didn't know. I changed because of default encode parameter is 'utf-8'. 
https://docs.python.org/3.6/library/stdtypes.html#str.encode